build.py

```diff
@@ -38,6 +38,7 @@
 use_plugin("source_distribution")
 
 use_plugin("python.unittest")
+use_plugin("python.pytest")
```

Comment:
If you need an integration test for the embedded plugin there are integration tests:

https://github.com/pybuilder/pybuilder/tree/master/src/integrationtest/python